conformance/operator_test.go

```diff
@@ -37,7 +37,10 @@ var _ = Describe("operator", func() {
 	})
 
 	BeforeEach(func() {
-		err := namespaces.Clean(operatorNamespace, namespaces.Test, clients)
+		isClusterReady, err := cluster.IsClusterStable(clients)
```

Comment:
Looks like we are in the right direction, but in case of unschedulable nodes this will make all tests fail instead of waiting for having them back.
Can you make this an Eventually, or merge it together with `SriovStable`?